### ISSUE TITLE
Compatibility with CP 1.10.0.rc.1

### ISSUE THREAD

**Issue by Brett-Best** [CocoaPods/cocoapods-acknowledgements#59]
Create a Gemfile:
```ruby
# frozen_string_literal: true

source "https://rubygems.org"

git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

gem "cocoapods", "1.10.0.rc.1"
gem "cocoapods-acknowledgements", "1.2.0"
```
Run:
`bundle install`

Error:
```
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    cocoapods-acknowledgements (= 1.2.0) was resolved to 1.2.0, which depends on
      activesupport (>= 4.0.2, < 5)

    cocoapods (= 1.10.0.rc.1) was resolved to 1.10.0.rc.1, which depends on
      cocoapods-core (= 1.10.0.rc.1) was resolved to 1.10.0.rc.1, which depends on
        activesupport (> 5.0, < 6)
```

Potential Fix:
https://github.com/Brett-Best/cocoapods-acknowledgements/commit/da66ba8639283dbce0fe5d83982a21ba02c0fa6b

**Comment by igor-makarov** [comment]
@dnkoutso I think this was fixed by #58 but no release was made since?

**Comment by Brett-Best** [comment]
Oh missed that it wasn’t in the tagged version, this works:`gem "cocoapods-acknowledgements", :git => "https://github.com/Cocoapods/cocoapods-acknowledgements.git", :branch => “master"`
